src/SplitIO/Grammar/Split.php

```diff
@@ -128,12 +128,19 @@ public function getTreatments()
     {
         $treatments = array();
 
-        if ($this->conditions) {
-            $condition = $this->conditions[0];
-            $treatments = $condition->getTreatments();
+//        if ($this->conditions) {
+//            $condition = $this->conditions[0];
+//            $treatments = $condition->getTreatments();
+//        }
+
+        foreach ($this->conditions as $condition) {
```

Comment:
@mredolatti please add the if condition before forarch: `if ($this->conditions) {`. It is to prevent warning messages at php log.